fdbserver/CommitProxyServer.actor.cpp

```diff
@@ -875,6 +890,11 @@ ACTOR Future<Void> preresolutionProcessing(CommitBatchContext* self) {
 	return Void();
 }
 
+int64_t getTenantIdFromMutation(const MutationRef& m) {
+	StringRef prefix = m.param1.substr(0, 8);
```

Comment:
Can we assert that param1 is at least 8 bytes? Additionally, if we are assuming that only one tenant is impacted can we also assert that?